health/REFERENCE.md

```diff
@@ -164,6 +166,30 @@ For example, you can create a template on the `disk.io` context, but filter it t
 families: sda sdb
 ```
 
+#### Alarm line `module`
+
+The `module` line, filters which module within the context this alarm should apply to. The value is a space-separated 
+list of simple patterns. See our [simple patterns docs](../libnetdata/simple_pattern/) for some examples.
+
+For example, you can create a template on the `isc_dhcpd.utilization` chart, but filter it to only the `isc_dhcpd` 
+module:
```

Comment:
it looks like this example doesn't really make sense

> you can create a template on the `isc_dhcpd.utilization`

it is already `isc_dhcpd`, there is not need to use `module: isc_dhcpd`

I dont see a good  example for just a `module` option, i think you'd better to show plugin + module

Reply:
And this is not really useful for users, this is for us because we are doing migration. Probably we dont even need to add this parameters to the documentation. @joelhans ? @vlvkobal ?

Reply:
I understand your point of view and I also would like to hear more comments about this before to do any change, because in theory we could use one or other, but for our use case it makes sense only both together.

Reply:
Reviewers please, we need answers here, because this can simplify our algorithm, on the other hand, I do not want to bring limitations for users.

Reply:
I do feel like we need to document them, as users might stumble across these lines if they're looking and health configuration files, even if they're only for our usage.

Perhaps we combine them as @ilyam8 suggested at first, and add a note that they're not something users should worry about. This is probably not totally correct, but as an example:

```markdown
> Note: The `plugin` and `module` lines are used internally by the Netdata team to warn users of deprecated collectors. Because health entities are already attached to a specific context, there is little real-world value in using these options.
```

Reply:
I think this is the only way to do it if you want to give it at least some sense. I see no req to use only **the** parameter in the parameter example.

Reply:
@joelhans I updated the documentation.
@ilyam8 it is only possible to use `plugin` and `module` together, I removed any possibility to use them independent.

Reply:
This was fixed yesterday.